diff --git a/lib/formatGroupedChanges.js b/lib/formatGroupedChanges.js
index 55710cad43f465cb2ddb60abcd05b1c164aa3e11..aa1c98254dd3123ce48c362842eca09437bda01e 100644
--- a/lib/formatGroupedChanges.js
+++ b/lib/formatGroupedChanges.js
@@ -1,16 +1,16 @@
 const formatGroupedChanges = (groupedChanges, locale, customTexts) => {
     let result = '';
     if (groupedChanges.added.length) {
-        result += `${customTexts.addedTitle || locale.addedTitle}\n` + groupedChanges.added.join('\n') + '\n\n';
+        result += `${customTexts.addedTitle || locale.addedTitle}\n` + groupedChanges.added.join('\n') + '\n';
     }
     if (groupedChanges.deleted.length) {
-        result += `${customTexts.deletedTitle || locale.deletedTitle}\n` + groupedChanges.deleted.join('\n') + '\n\n';
+        result += `${customTexts.deletedTitle || locale.deletedTitle}\n` + groupedChanges.deleted.join('\n') + '\n';
     }
     if (groupedChanges.edited.length) {
-        result += `${customTexts.editedTitle || locale.editedTitle}\n` + groupedChanges.edited.join('\n') + '\n\n';
+        result += `${customTexts.editedTitle || locale.editedTitle}\n` + groupedChanges.edited.join('\n') + '\n';
     }
     if (groupedChanges.arrayChange.length) {
-        result += `${customTexts.arrayChangeTitle || locale.arrayChangeTitle}\n` + groupedChanges.arrayChange.join('\n') + '\n\n';
+        result += `${customTexts.arrayChangeTitle || locale.arrayChangeTitle}\n` + groupedChanges.arrayChange.join('\n') + '\n';
     }
     
     return result.trim();
